api/database/seeders/PoolCandidateTestSeeder.php

```diff
@@ -79,6 +81,7 @@ public function run()
                 'email' => 'try-hard@test.com',
                 'sub' => 'try-hard@test.com',
                 'citizenship' => CitizenshipStatus::PERMANENT_RESIDENT->name,
+                'armed_forces_status' => null,
```

Comment:
Not a veteran but complete profile should set it to `ArmedForcesStatus::NON_CAF->name`

Reply:
Oh, I did not realize that :sweat_smile: Fixed in [fix non-veteran candidate seeder](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10562/commits/d6dd483616716b1961335361d154ce5923dd0e7e)